build/DependencyVersions.props

```diff
@@ -11,13 +11,16 @@
     <SharedFrameworkVersion>$(CLI_SharedFrameworkVersion)</SharedFrameworkVersion>
     <SharedHostVersion>$(CLI_SharedFrameworkVersion)</SharedHostVersion>
     <HostFxrVersion>$(CLI_SharedFrameworkVersion)</HostFxrVersion>
-    <TemplateEngineVersion>1.0.0-beta2-20170427-205</TemplateEngineVersion>
-    <TemplateEngineTemplateVersion>1.0.0-beta2-20170502-215</TemplateEngineTemplateVersion>
-    <TemplateEngineTemplate2_0Version>1.0.0-beta2-20170502-215</TemplateEngineTemplate2_0Version>
+    <TemplateEngineVersion>1.0.0-beta2-20170503-217</TemplateEngineVersion>
+    <TemplateEngineTemplateVersion>1.0.0-beta2-20170503-217</TemplateEngineTemplateVersion>
+    <TemplateEngineTemplate2_0Version>1.0.0-beta2-20170503-217</TemplateEngineTemplate2_0Version>
     <PlatformAbstractionsVersion>2.0.0-preview1-002106</PlatformAbstractionsVersion>
     <DependencyModelVersion>2.0.0-preview1-002106</DependencyModelVersion>
     <CliCommandLineParserVersion>0.1.0-alpha-142</CliCommandLineParserVersion>
-    <AspNetCoreRuntimeVersion>2.0.0-preview1-56</AspNetCoreRuntimeVersion>
+    <AspNetCoreRuntimeVersion>2.0.0-preview1-65</AspNetCoreRuntimeVersion>
+
+    <!-- This should either be timestamped or notimestamp as appropriate -->
+    <AspNetCoreRuntimePackageFlavor>timestamped</AspNetCoreRuntimePackageFlavor>

```

Comment:
Why would we keep changing this? And why do we want to ship preview1 with the timestamped one?